plugin/pkg/admission/noderestriction/admission.go

```diff
@@ -241,6 +250,40 @@ func (c *nodePlugin) admitPodEviction(nodeName string, a admission.Attributes) e
 	}
 }
 
+func (c *nodePlugin) admitPVCStatus(nodeName string, a admission.Attributes) error {
```

Comment:
If you aren't performing any mutation, you should do this in the `Validate` method of admission.

Reply:
> If you aren't performing any mutation, you should do this in the Validate method of admission.

NM, I was mentally in the wrong plugin.